src/App.js

```diff
@@ -74,7 +75,7 @@ const App = ({ vms, visibility, config }) => {
   return (
     <Router getUserConfirmation={openConfirmation} basename={AppConfiguration.applicationURL}>
       <div>
-        <VmsPageHeader page={vms.get('page')} title={fixedStrings.BRAND_NAME + ' VM Portal'} />
+        <VmsPageHeader page={vms.get('page')} title={fixedStrings.BRAND_NAME + ' ' + msg.vmPortal()} />
```

Comment:
I believe it's ok for this particular case, but in general we should avoid such string composition due to different word order in various languages.